src/AppBundle/Entity/Report/Traits/StatusTrait.php

```diff
@@ -26,6 +26,20 @@
      */
     private $statusCached;
```

Comment:
yes it should be renamed. I had to limit the amount of refactors or this PR will never end :D.
I was actually considering removing the section status. Since we now store the final status, the section status is basically useless. It can be fully calculated in a few ms from the API at each report edit operation with a small overhead and much less code